open-api/rest-catalog-open-api.yaml

```diff
@@ -532,6 +532,103 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/namespaces/{namespace}/tables/{table}/preplan:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+      - $ref: '#/components/parameters/namespace'
+      - $ref: '#/components/parameters/table'
+    post:
+      tags:
+        - Catalog API
+      summary: Find plan-tasks based on a query.
+      description:
+        When a user submits a query, this operation will find the relevant plan-tasks
+        based on the user's selected columns, and filters. The plan-tasks can be later used during "PlanTable"
+        to distribute this work for performance gain.
+      operationId: PreplanTable
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PreplanTableRequest'
```

Comment:
I believe case does not match here and it's `PrePlanTableRequest` below at line 3099,

Reply:
Thanks for the catch will fix this